### PR TITLE
Heroku different app error page

### LINKED ISSUE #135: Article about troubleshooting different app on Heroku errors
Sometimes customers say that they have pointed their domain to their Heroku app and that they get an app that isn't theirs (or is theirs but is not the one they wanted). 

This article would be similar to:

- http://support.dnsimple.com/articles/heroku-error-ssl/
- http://support.dnsimple.com/articles/heroku-error-nosuchapp/

### PR DESCRIPTION
**Pull request by jacegu**
Closes #135 

This one also includes a link to a section that is not in master yet.

### CODE REVIEW

File: content/articles/heroku-error-differentapp.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+excerpt: You pointed your domain to Heroku, but when you access your domain you see a different application than the one you expected.
+categories:
+- Heroku and DNSimple
+---
+
+# Troubleshooting Heroku shows different app error
+
+You pointed your domain to Heroku, but when you access your domain from the browser you see a different application than the one you expected.
+
+This error occurs when you point a custom domain to Heroku, but there is another app at Heroku with that domain name set up as a custom domain.
```
Comment (weppos) [inline content/articles/heroku-error-differentapp.markdown]: This sentence is a little bit unclear. I'd rephrase it to

> This error occurs when in DNSimple you point the DNS for a domain to Heroku, but in Heroku you associated the custom domains to a different application.

I'd also add an example

> For example, you have two Heroku applications: `green-turtle` and `red-turtle` and the corresponding Heroku hostnames `green-turtle.herokuapp.com` and `red-turtle.herokuapp.com`. You add the domain `awesome-turtles.com` in DNSimple and you point the CNAME `www.awesome-turtles.com` to `green-turtle.herokuapp.com`. However, in Heroku you associate the `www.awesome-turtles.com` custom domain to the application `red-turtle.herokuapp.com`.
> What will happen is that when someone will go to `www.awesome-turtles.com`, Heroku will route the request to the `red-turtle` app, and not `green-turtle` as you would expect.

**Comment by weppos** [comment]
I generally use `curl` to debug the issue. I send an HTTP request to the expected endpoint and I append a custom `Host` header.

```
curl -I -H 'Host: www.awesome-turtle.com' http://green-turtle.herokuapp.com
```

Then I either perform the same request via curl again without the custom header, or I visit the site and I compare if they match.

```
curl -I http://green-turtle.herokuapp.com
```

If they don't, it means there is a routing issue. Does it make sense to add the first curl example as a possible debug hint?

File: content/articles/heroku-error-differentapp.markdown (hunk 1 above)
Comment (jacegu) [inline content/articles/heroku-error-differentapp.markdown]: Adding this as I find it very useful. 

Is there some syntax that I can use to make it clear that it's an example and not a text explanation at the same level of everything else?

**Comment by jacegu** [comment]
I'd like the example to differentiate from the rest of the text. Not sure what syntax to use there. Ideas?

**Comment by weppos** [comment]
We don't have a specific syntax for examples at the moment.

**Comment by jacegu** [comment]
@weppos I'm not sure whether it's worth it to include the curls here. If they end up in this page they already know the displayed app is not what they expected which is what the curl will help them figuring out.

What do you think?

**Comment by weppos** [comment]
:+1:

**Comment by jcaudle** [comment]
I put some phrasing tweaks in, but this looks pretty good. I had no idea this could actually happen. :scream_cat:

**Comment by jacegu** [comment]
:ship: this ?

**Comment by weppos** [comment]
giphy
